src/Filesystem/README.md

```diff
@@ -0,0 +1,377 @@
+# Filesystem Service
```

Comment:
Thank you so much for the Readme! Loving it. You might think about adding a TOC. Note that I myself did not do that so far in own Readme's but believe they might be useful and try to improve :-)